chatroom/src/resources/welcomeScreen/config.json

```diff
@@ -0,0 +1,20 @@
+[
+    {
+      "cardTitle": "WebDev",
+      "id": null,  
```

Comment:
id refers to some "primary key" of each element. You may use 1, 2, 3, etc. for the elements.